includes/User.php

```diff
@@ -4218,6 +4218,20 @@ public static function crypt( $password, $salt = false ) {
 	 * @return Boolean
 	 */
 	public static function comparePasswords( $hash, $password, $userId = false ) {
+		// Wikia change - begin
+		// @see PLATFORM-2502 comparing new passwords in PHP code.
+		// @todo mech remove after the new password hashing is implemented.
```

Comment:
I would suggest create follow up ticket to cleanup the code so we won't loose it when this gets reassigned to someone else